api/controllers/SiteController.js

```diff
@@ -6,7 +6,9 @@
  */
 
 module.exports = {
-
+  delete: function(req, res) {
```

Comment:
@el-mapache Is this function still needed here? I think we get a delete action for free through a feature of Sails called blueprint routes